src/main/java/nl/novi/serviceconnect/helpper/Helpers.java

```diff
@@ -5,5 +5,7 @@
 
 public class Helpers {
     public static boolean isNotNullOrEmptyDate(Date date) { return date != null && !date.equals(new Date(0)); }
-    public static boolean isValidFile(File file) { return file != null && file.exists() && file.isFile(); }
+    public static boolean isNotNullOrEmpty(String value) {
```

Comment:
Suggestie: Geen dubbele ontkenning gebruiken. Het is logischer te begrijpen met een IsNullOrEmpty() functie.

Reply:
done